### PR TITLE
Add Prometheus to collect and view ContainerPilot telemetry

### PR DESCRIPTION
**Pull request by misterbisson**
This PR adds Prometheus to the Docker Compose file so we can easily demo ContainerPilot telemetry in this composition.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -76,24 +98,15 @@ nginx:
...
-consul:
-  image: autopilotpattern/consul:latest
-  command: >
-    /usr/local/bin/containerpilot
-    /bin/consul agent -server
-    -bootstrap
-    -config-dir=/etc/consul
-    -ui-dir /ui
+# Prometheus is an open source performance monitoring tool
+# it is included here for demo purposes and is not required
```
Comment (tgross) [inline docker-compose.yml:102]: We're not adding this to the local-compose.yml as well?

File: docker-compose.yml [hunk 2]
```diff
@@ -76,24 +98,15 @@ nginx:
...
+prometheus:
+  image: autopilotpattern/prometheus:latest
   restart: always
-  mem_limit: 128m
+  mem_limit: 1g
   env_file: _env
```
Comment (tgross) [inline docker-compose.yml:107]: Not for this PR, but I wonder if we should try to come up with a better way to inject the CONSUL env var than this. Right now we're feeding the MySQL and WP creds to all the containers.

**Comment by tgross** [comment]
Just the one actionable comment about local-compose.yml but otherwise LGTM.

File: docker-compose.yml (hunk 1 above)
Comment (misterbisson) [inline docker-compose.yml:102]: Now added.

File: docker-compose.yml (hunk 2 above)
Comment (misterbisson) [inline docker-compose.yml:107]: Agreed.

**Pull request merged**